pkg/controller/deorbit/metrics.go

```diff
@@ -0,0 +1,53 @@
+package deorbit
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	core_v1 "k8s.io/api/core/v1"
+)
+
+type InstrumentingDeorbiter struct {
+	Deorbiter Deorbiter
+
+	Latency    *prometheus.SummaryVec
+	Total      *prometheus.CounterVec
+	Successful *prometheus.CounterVec
+	Failed     *prometheus.CounterVec
+}
+
+func (d *InstrumentingDeorbiter) DeletePersistentVolumeClaims() (deleted []core_v1.PersistentVolumeClaim, err error) {
+	defer d.instrument("DeletePersistentVolumeClaims", time.Now(), err)
+	return d.Deorbiter.DeletePersistentVolumeClaims()
+}
+
+func (d *InstrumentingDeorbiter) DeleteServices() (deleted []core_v1.Service, err error) {
+	defer d.instrument("DeleteServices", time.Now(), err)
+	return d.Deorbiter.DeleteServices()
+}
+
+func (d *InstrumentingDeorbiter) WaitForPersistentVolumeCleanup() (err error) {
+	defer d.instrument("WaitForPersistentVolumeCleanup", time.Now(), err)
+	return d.WaitForPersistentVolumeCleanup()
```

Comment:
Should be`d.Deorbiter.WaitForPersistentVolumeCleanup()`

Reply:
eagle eye. fixed